src/styles/utils/_colors.scss

```diff
@@ -1,7 +1,16 @@
 $libraryRed: #E32B31 !default;
 $libraryBlue: #1B7FA7 !default;
 $BG_MENU_SEARCH: #1B7FA7;
+
 $nypl-teal: #07818d;
+$nypl-orange: #ffb81d;
```

Comment:
These are not in the Design Toolkit?

Reply:
they are... 
see:
https://github.com/NYPL/design-toolkit/blob/f21d84d7fa4c32fa994f2b06ee2285b0033274e1/sass/_colors.scss#L41

and 

https://github.com/NYPL/design-toolkit/blob/f21d84d7fa4c32fa994f2b06ee2285b0033274e1/sass/_colors.scss#L26

respectively. Is there an issue here?

Reply:
If they're in the DT, then they don't need to be re-declared here?

Reply:
🤔 this application doesnt use the DTK ¿que no?

Reply:
@EdwinGuzman I think I'll need a short tutorial on how exactly dgx-header uses the DTK...

Reply:
It doesn't until you added it just now. So I believe those variables `$nypl-orange` should be available without having to re-declare them.